### PR TITLE
Added SQL quiz question for sql-quiz.ts

### PR DESCRIPTION
**Pull request by devesh-mishra13**
## Summary of changes

Changes made for sql-quiz.ts by adding a quiz question related to BITWISE AND.

## Checklist
- [x] I have read and followed the [contribution guidelines](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CONTRIBUTING.md).
- [x] I have read through the [Code of Conduct](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CODE_OF_CONDUCT.md) and agree to abide by the rules.
- [x] This PR is for one of the available issues and is not a PR for an issue already assigned to someone else.
- [x] My PR title has a short descriptive name so the maintainers can get an idea of what the PR is about.
- [x] I have provided a summary of my changes.

### CODE REVIEW

File: src/data/sql-quiz.ts [hunk 1]
```diff
@@ -831,26 +831,38 @@ const sqlQuiz = [
...
     Distractor1: "to store data for a short-lived analysis such as 1-2 weeks",
     Distractor2: "to improve database security",
     Distractor3: "to improve query performance",
     Explanation:
       "A temporary SQL table is a table that is created and used within the context of a specific session or transaction in a database.",
     Link: "https://www.freecodecamp.org/news/sql-temp-table-how-to-create-a-temporary-sql-table/"
+  },
+  {
+    Question: "What does the bitwise AND operator do in SQL?",
+    Answer: "Performs AND operation on corresponding bits of two numbers.",
```
Comment (jdwilkin4) [inline src/data/sql-quiz.ts:859]:
```suggestion
    Answer: "Performs a logical AND operation on corresponding bits of two numbers.",
```

File: src/data/sql-quiz.ts [hunk 2]
```diff
@@ -831,26 +831,38 @@ const sqlQuiz = [
...
+    Distractor1: "Combines two strings.",
+    Distractor2: "Adds two numbers.",
+    Distractor3: "Multiplies two numbers.",
+    Explanation:
+      "In SQL, the operation 'BITWISE AND' is used to perform a bitwise AND operation between two values. This operation compares each bit of the two values and sets the corresponding bit in the result to 1 only if both bits are 1.",
+    Link: "https://www.freecodecamp.org/news/learn-sql-in-10-minutes/"
```
Comment (jdwilkin4) [inline src/data/sql-quiz.ts:865]: this article goes more in depth

```suggestion
    Link: "https://www.freecodecamp.org/news/sql-operators-tutorial/#bitwise-operators"
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

File: src/data/sql-quiz.ts (hunk 2 above)
Comment (devesh-mishra13) [reply src/data/sql-quiz.ts:865]: thank You for your suggesion but do i just have to Commit the suggesion that you have provided me.

**Comment by devesh-mishra13** [comment]
i closed it by mistake what can i do now i was making another commit and during merging of a branch i actually merged the same branch before even pulling the changes please help me

**Comment by jdwilkin4** [comment]
@devesh-mishra13 
It looks like I am not able to reopen your PR on my end because there is already another open PR here for the same branch of main

https://github.com/freeCodeCamp/Developer_Quiz_Site/pull/786

Here is what  I suggested you do.

Add this SQL question with the revised changes to your new open PR here
https://github.com/freeCodeCamp/Developer_Quiz_Site/pull/786

Then when it is reviewed, approved and merged, then you will need to update your main branch.

Then if you are interested in contributing more quiz questions after that, then I would suggest create a new branch that is not the main branch.
It is common practice in open source to create new branches that are not the main branch when making new contributions.
It makes for a cleaner process that way and keeps your main branch clean and easy to work with.

Here is a more in depth tutorial on git branches which will help in the future
https://www.youtube.com/watch?v=e2IbNHi4uCI

Hope that helps 👍

**Comment by devesh-mishra13** [comment]
okay thank you for your suggestions i would be taking atmost care the next time

**Pull request closed without merging**